sql/src/test/java/io/crate/execution/dml/upsert/UpdateSourceGenTest.java

```diff
@@ -166,9 +166,9 @@ public void testNestedGeneratedColumnIsGenerated() throws Exception {
     @Test
     public void testNestedGeneratedColumnIsGeneratedValidateValueIfGivenByUser() throws Exception {
         SQLExecutor e = SQLExecutor.builder(clusterService)
-            .addTable("create table t (x long, obj object as (y as x + 1))")
+            .addTable("create table t (x int, obj object as (y as 'foo'))")
```

Comment:
Not sure why but these 2 tests were failing randomly - when they did fail, the generated expression i.e. `input.value()` resulted in `null`.
Any idea why?

It seems to be a glitch with the test itself, I have verified with an integration test that it is not an issue with the current
changes.

Reply:
No clue, but I think we've to investigate this instead of changing the tests.